lib/config.js

```diff
@@ -73,11 +73,14 @@ module.exports = function (options, configPath) {
     meta = require(metaPath);
 
     if (meta.sasslintConfig) {
-
       configPath = path.resolve(path.dirname(metaPath), meta.sasslintConfig);
     }
     else {
       configPath = findFile(false, '.sass-lint.yml');
+
+      if (!configPath) {
+        configPath = findFile(false, '.sassrc');
+      }
```

Comment:
Tests `.sass-lint.yml` prior to `.sasslintrc`, as most users will still be using `.sass-lint.yml`.